.prettierignore

```diff
@@ -40,6 +40,7 @@ app/ide-desktop/lib/dashboard/dist/
 app/gui/view/documentation/assets/stylesheet.css
 app/rust-ffi/pkg
 app/gui/src/project-view/assets/font-*.css
+app/gui/mockDist
```

Comment:
Fixed differently in https://github.com/enso-org/enso/pull/11317/commits/e0310e57f7cc62bb52b8d124f78d8802fd53ff30

I think it is closer to built artifacts than generated files